### DISCUSSION TITLE
New badges?

### DISCUSSION THREAD

**Discussion by perezident14** [community/community#18315] [Discussions] [answered]
How do you unlock the new badges and are there more than just the 3 from the blog post?

**Comment by p14** [comment, marked as answer]
Galaxy Brain: 2 discussions answered
Pull Shark: 2 PRs merged
YOLO: Merge a PR without a review

**Reply by dhanisetiaji** [reply]
thanks

**Comment by baoloc7401** [comment]
Here's a whole FYI: https://github.com/Schweinepriester/github-profile-achievements

**Comment by palagdan** [comment]
Galaxy Brain: 2 discussions answered

**Comment by tanny99** [comment]
Please accept this answer,
I want the badgeee
pleaseeee
